packages/@react-aria/interactions/src/useFocusVisible.ts

```diff
@@ -89,7 +89,9 @@ function handleFocusEvent(e: FocusEvent) {
   // Firefox fires two extra focus events when the user first clicks into an iframe:
   // first on the window, then on the document. We ignore these events so they don't
   // cause keyboard focus rings to appear.
-  if (e.target === window || e.target === document) {
+  const windowObject = getOwnerWindow(e.target as HTMLElement);
+  const documentObject = getOwnerDocument(e.target as HTMLElement);
+  if (e.target === windowObject || e.target === documentObject) {
```

Comment:
are these lines actually equivalent to the original? This logic is specifically for iframes to begin with and now we're comparing to a different window/document than before I think?

were you able to reproduce the events that the code comment calls out?